docs/releases/5.2.txt

```diff
@@ -466,3 +471,11 @@ Miscellaneous
 * The fallback to ``request.user`` when ``user`` is ``None`` in
   ``django.contrib.auth.login()`` and ``django.contrib.auth.alogin()`` will be
   removed.
```

Comment:
The depredations need to be added to `docs/internals/deprecation.txt`

Reply:
I have added some documentation around the items to be removed. Please let me know if it meets the needs/standards.